### PR TITLE
Fix bad jenkins "pytests" runs

### PR DESCRIPTION
**Pull request by portante**
And as a result, we fix the issues caught by these fixes.

### CODE REVIEW

**Review by dbutenhof — APPROVED** [review]
Cool!

File: jenkins/run [hunk 1]
```diff
@@ -54,6 +54,9 @@ if [[ ${?} -ne 0 ]]; then
     exit 1
 fi
 
+find . -name __pycache__ -exec rm -r {} \; > /dev/null 2>&1
```
Comment (npalaska) [inline jenkins/run:57]: should we do `-name __pycache__ -or -name '*.pyc'` I find sometime I need to clear .pyc files to get the pytest working, but if they also get clear when we clear the cache thats fine

Comment (portante) [reply jenkins/run:57]: Where do you find the `.pyc` files?

Comment (npalaska) [reply jenkins/run:57]: Ahh interesting I didnt notice they all fall under `__pycache__` directory, cool

**Review by npalaska — APPROVED** [review]

**Review by ndokos — APPROVED** [review]

**Pull request merged**